src/types/d2-api.ts

```diff
@@ -0,0 +1,2 @@
+export * from "@eyeseetea/d2-api/";
+export * from "@eyeseetea/d2-api/2.36";
```

Comment:
Shouldn't we be using 2.37? Not even sure if we already have the infrastructure in place in d2-api though. @tokland ?

Reply:
We have up to 2.36, I'll create a PR in d2-api for 2.37 (thought the types used in the script are not affected)